cabot_alert_pagerduty/models.py

```diff
@@ -25,9 +26,19 @@ class PagerdutyAlert(AlertPlugin):
     name = "Pagerduty"
     author = "Mahendra M"
 
+    @property
+    def alert_status_list(self):
+        if not hasattr(self, '_alert_status_list'):
```

Comment:
We don't seem to ever explicitly set `_alert_status_list` anywhere, can we just forgo the attribute check and check `_gather_alertable_status()` every time?